en_US/data-integration/rules.md

```diff
@@ -30,72 +52,98 @@ WHERE
     clientid = "foo"
 ```
 
-In the above SQL:
+The above SQL statement specifies the following for the rule:
 
-- Data Source: the messages with topic `t/#`;
-- Data Processing: If the client ID of the message is `foo`, select the `data` field from the message content and assign it to the new variable `d`
+- **Data source**: Messages with topic `t/#` and client ID as `foo`
+- **Data transformation**: Extract the `data` field from the payload and assign a new field `d`. So in the above example, the output data will have the following structure `{d: "value of the payload's data field"}`.
```

Comment:
Extract the `data` field from the payload and assign a new field `d`. -> Rename the field named `data` field to `d`.